__tests__/app.test.js

```diff
@@ -23,12 +24,28 @@ describe("nc-news", () => {
         .get("/api/topics")
         .expect(200)
         .then(({ body }) => {
-            const { topics } = body
+          const { topics } = body;
           expect(topics.length).toBe(3);
           topics.forEach((topic) => {
-            expect(typeof topic.slug).toBe('string')
-            expect(typeof topic .description).toBe('string')
-          })
+            expect(typeof topic.slug).toBe("string");
+            expect(typeof topic.description).toBe("string");
+          });
+        });
+    });
+  });
+
+  describe("GET /api", () => {
+    test("GET 200: returns json object listing all endpoints with information about them", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.endpoints).toMatchObject({
+            "GET /api": {
+              description:
+                "serves up a json representation of all the available endpoints of the api",
+            },
```

Comment:
This isn't the most flexible test, as you may want to change your descriptions of endpoints in the future, so hardcoding could cause your test to fail. You could change the test to read your .json file in the same way you use your controller. The testing of this endpoint would always be quite brittle, so we're not too worried about it :)